app.rb

```diff
@@ -1,9 +1,29 @@
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
```

Comment:
For this route, you could redirect to `/peeps` or show a page with a button which takes you to `/peeps`

Reply:
This would probably be a better UX